src/scancode/cli.py

```diff
@@ -467,6 +469,26 @@ def scan_event(item):
     return files_count, cached_scan.iterate(scans, root_dir)
 
 
+def shorten_filename(filename='', max_length=25, number_of_dots=3):
+    """
+    Return the original filename if it is short enough
```

Comment:
Or you can test ass an exercise that the proper exception is raised in this case.